### PR TITLE
remove past shows

### PR DESCRIPTION
**Pull request by crgrafton**
RHPS with The Bawdy Caste!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,10 +91,13 @@ <h3 class="card-title">Oct 4th</h3>
...
+
             </div>
+
             <div class="row">
-                <div class="col">
+
+                <!-- <div class="col">
                     <div class="card">
                         <img class="card-img-top "
                             alt=wewantedtopostthepictureofthecastfromtwoyearsago,butithasaverysexyphotoofwill/alex/lexiandyouknow,cucks,soitsjusttherenosignfromapublicdomainsite
```
Comment (crgrafton) [inline index.html]: @jaison2000b this is here, in the history of our site, forever...

Don't do things like this in the future. It's unkind, unprofessional, and this cast expects better of you.

Comment (jaison2000b) [reply index.html]: Should've got in the review, also shouldn't have restored site to original Oct, bullshit, when you are in a bad mood Don't do things like this in the future. It's unkind, unprofessional, and this cast expects better of you.

**Comment by jaison2000b** [comment]
few days late but better late than never

File: index.html (hunk 1 above)
Comment (crgrafton) [reply index.html]: I can't really understand what you've written, but your childish antics aren't appreciated.

**Pull request merged**